crates/bevy_input/src/input.rs

```diff
@@ -120,7 +120,6 @@ where
     pub fn clear(&mut self) {
         self.just_pressed.clear();
         self.just_released.clear();
-        self.pressed.clear();
```

Comment:
can you leave a comment hear about why pressed is not called, so this doesn't get added again?

Reply:
I forgot to revert the test change. I added a comment in the test